dev/DynamicDependency/M.AM.DD.CreatePackageDependencyOptions.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "M.AM.DD.CreatePackageDependencyOptions.h"
+#include "Microsoft.ApplicationModel.DynamicDependency.CreatePackageDependencyOptions.g.cpp"
```

Comment:
[External]
So this one doesn't get shortened, but the others do? 🙂

Reply:
Reference to a generated file. Only the manually created + edited files are named with the abbreviation